playbook/app/pb_kits/playbook/pb_bread_crumbs/_bread_crumb_item.tsx

```diff
@@ -37,7 +37,7 @@ const BreadCrumbItem = (props: BreadCrumbItemProps): React.ReactElement => {
     globalProps(props),
     className
   )
-  
+
```

Comment:
To get rid of this extra line space, you can either remove it, and add, commit, and push or you can run this:
```sh
git checkout origin/master -- playbook/app/pb_kits/playbook/pb_bread_crumbs/_bread_crumb_item.tsx
```